src/AccessControlConnector/Connectors/VillaProConnector.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Innovatrics.SmartFace.Integrations.AccessControlConnector.Models;
+using Serilog;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+using Innovatrics.SmartFace.Integrations.AccessControlConnector.Factories;
+using Innovatrics.SmartFace.Models.API;
+using Innovatrics.SmartFace.Integrations.AccessControlConnector.Resolvers;
+
+namespace Innovatrics.SmartFace.Integrations.AccessControlConnector.Connectors
+{
+    public class VillaProConnector : IAccessControlConnector
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+        private readonly IHttpClientFactory httpClientFactory;
+
+        private string authToken;
+        private string systToken;
+        private string baseUrl;
+
+        public VillaProConnector(ILogger logger, IConfiguration configuration, IHttpClientFactory httpClientFactory)
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
```

Comment:
related to previous column "this." should be omitted as well unless explictly needed due to inheritance